app/controllers/crons_controller.rb

```diff
@@ -5,7 +5,10 @@ def start
     user_id = "fillIn"
     access_token = "fillIn"
     date = Time.now.strftime("%Y/%m/%d") # date = YYYY/MM/DD
-    res = RestClient.get("https://api.fitbit.com/1.2/user/#{user_id}/sleep/date/#{date}.json", headers={authorization: "Bearer #{access_token}"})
-    p res.to_yaml
+    res = JSON.parse(RestClient.get("https://api.fitbit.com/1.2/user/#{user_id}/sleep/date/#{date}.json", headers={authorization: "Bearer #{access_token}"}))
+    
+    if (res.key?(:meta)) # res should have a meta key when sleep logs are processing (requested during a sleep still ongoing) see "metaresponse" in https://dev.fitbit.com/build/reference/web-api/sleep/get-sleep-log-by-date/
+      p "USER IS ASLEEP RIGHT NOW"
+    end
```

Comment:
Can we add another log if this ran but there was no meta tag? Otherwise we will not be able to tell if it ran to the end

It may also be useful to print the entire response every time